pkg/sqs/SqsConnectionFactory.php

```diff
@@ -38,7 +39,9 @@ class SqsConnectionFactory implements PsrConnectionFactory
      */
     public function __construct($config = 'sqs:')
```

Comment:
I think the constructor should accept an instance of Client, no need to pass it as an array parameter. We would not be able to apply other options in the array in any case.

Reply:
The symfony transport should have it as a client option though

Reply:
Done.